static/js/import_wizard/utils/validation.ts

```diff
@@ -60,27 +62,40 @@ export function checkMappings(mappings: Mapping): Array<string> {
   let numNonConsts = 0;
   mappings.forEach((mval: MappingVal, mthing: MappedThing) => {
     const mthingName = MAPPED_THING_NAMES[mthing] || mthing;
-    if (mthing === MappedThing.PLACE) {
-      if (
-        mval.placeProperty == null ||
-        mval.placeProperty.dcid == null ||
-        mval.placeProperty.dcid === ""
-      ) {
-        // Check #6
-        errors.push("Place mapping is missing placeProperty");
-      }
-    }
     if (mval.type === MappingType.COLUMN) {
       if (mval.column == null || mval.column.id === "") {
         // Check #1
         errors.push(mthingName + ": missing value for COLUMN type ");
       }
+      if (mthing === MappedThing.PLACE) {
+        if (
+          _.isEmpty(mval.placeProperty) ||
+          _.isEmpty(mval.placeProperty[mval.column.columnIdx])
```

Comment:
Also assert placeProperty Record has only one entry?

Reply:
I'm not sure this is necessary because when generating tmcf and observations, if the mapping type is COLUMN, we will only read the place property for the specified column. Also, there is only one place to input a place property and type when user has chosen mapping type column, so I don't think it'll be possible for there to be more than one entry

Reply:
If you consider the point of this function is to make correctness assertions on mappings based on the spec, without assuming any knowledge of code that updates or uses the mappings (that was me when I wrote it), how would you feel about this assertion? :)

If you're still unsure, feel free to skip...

Reply:
ok, I see your point, updated!

Reply:
nit: 
```
if (Object.keys(mval.placeProperty).length !== 1) {
   errors.push("Place mapping for a column expected to have one placeProperty");
}
```